src/components/RadioButton/__tests__/radioButton-tests.jest.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { fireEvent, render, cleanup, screen } from "@testing-library/react";
+import RadioButton from "../RadioButton.jsx";
+
+describe("RadioButton tests", () => {
+  const formName = "myForm";
+  const radiosName = "radios";
+  const option1Value = "1";
+  const option1Text = "Option 1";
+
+  const option2Value = "2";
+  const option2Text = "Option 2";
+
+  const option3Value = "3";
+  const option3Text = "Option 3";
+
+  let onChangedMock;
+
+  beforeEach(() => {
```

Comment:
Please add a test with `getByText` so we would make sure that both the `aria-label` and the text are in connection to the input element